specificatie/openapi.yaml

```diff
@@ -552,7 +552,7 @@ components:
         example: '0226100000008856'
 
     adresseerbaarObjectIdentificatie-query:
-      description: "De identificatie van een adresserbaar object uit de BAG. Dit kan een verblijfsobject, een standplaats of een ligplaats zijn."
+      description: "De identificatie van een adresseerbaar object uit de BAG. Dit kan een verblijfsobject, een standplaats of een ligplaats zijn."
```

Comment:
Bij de pad parameter vermeld je wel dat hij 16 lang is en hier niet. Dan zou ik het hier ook toevoegen.

Reply:
Ik ben het niet helemaal eens met Johan hier. Bij een parameter moet je vrij precies zijn in wat iemand moet opgeven, de gebruiker moet immers weten hoe de API gebruikt moet worden.
Bij een gegeven in het antwoord is dat veel minder belangrijk. De gebruiker moet gegevens gewoon gebruiken zoals zij/hij deze krijgt. Daarom hebben we volgens mij altijd als stelregel gehad dat we voor gegevens in het antwoord weinig precies specificeren (behalve wanneer dat echt nodig is zoals bij een enumeratie).

Reply:
Volgens mij heb ik deze opmerking bij de query-parameter geplaatst.

Reply:
sorry, je hebt gelijk. je opmerking is terecht